README.md

```diff
@@ -3,7 +3,14 @@
 Adds support for derived resources to a
 [Community Solid Server](https://github.com/CommunitySolidServer/CommunitySolidServer) instance.
 
-Run `npm start` to have a CSS instance that combines `@css:config/file.json` with the new components here.
+## Install
+
+```
+npm install
+npm run build
+```
+
+Run `npm run start:example` to have a CSS instance that combines `@css:config/file.json` with the new components implementing the template `templates/root/base`.
```

Comment:
This is already in the examples section of the readme so is it needed here? If you do think it would be better to also have it here, this is wrong though as the example script uses `default.json` instead of `file.json`.

Reply:
Fixed the `default.json`. 

About "already in the example section": it is my personal taste for documentation that starts with a short synopsis to get a demonstrator up and running and afterwards read about the details. The current README asks to run `npm run start` but this doesn't do anything visible unless I create all the derived resource myself. The `npm run start:example` is a nice synopsis, it has immediate results and shows what the code tries to solve.